consensus/rules/src/batch_verifier.rs

```diff
@@ -0,0 +1,24 @@
+use multiexp::BatchVerifier as InternalBatchVerifier;
+
+/// This trait represents a batch verifier.
+pub trait BatchVerifier {
```

Comment:
What is a batch verifier, why is it needed, what does it do, etc.

Reply:
added some docs